waku.nimble

```diff
@@ -31,6 +32,16 @@ requires "nim >= 2.2.4",
   "quic"
```

Comment:
```suggestion
```

`quic` is not being used directly by nwaku, we should let nim-libp2p bring the dependency in.